pyemu/utils/helpers.py

```diff
@@ -1689,7 +1689,17 @@ def _process_array_file(model_file, df):
                 lb = float(list(lb_vals.keys())[0])
                 org_arr[org_arr < lb] = lb
 
-    np.savetxt(model_file, np.atleast_2d(org_arr), fmt="%15.6E", delimiter="")
+    try:
+        fmt = df_mf.fmt.iloc[0]
+    except AttributeError:
+        fmt = "%15.6E"
```

Comment:
I guess we need to keep the 15-6 for old school modflow/mt3d?

Reply:
I think with PstFrom it should be ok. The detect 'should' get the right format. We need to is fall-back for the old pstfromflopy method which I don't think stores a 'fmt' (and I didn't fancy opening that box!)